### ISSUE TITLE
Add Method to visualizations?

### ISSUE THREAD

**Issue by christinabergmann** [langcog/metalab-archive#94]
Method (e.g., central fixation, high-amplitude sucking) is a more fine-grained and possibly informative descriptor than behavior/eye tracking (but nested within), so it might be useful to look at this one separately while keeping the option to just separate response modes. 

(Ideally, there would be radio buttons for method and response mode, but I think that's pushing the limits of shiny.)

**Comment by mcfrank** [comment]
Yes, I agree. The thing is, this is likely a very MA-specific feature, so
it's hard to integrate with all the other MAs that don't have that level of
detail. An alternative would be just to code HAS as its own method category
(and not call it "behavior").

Mike
